YvesIraguha/rhinos-app
# Add Apollo Client

### What does this PR do? 
- configures ApolloClient 

### How can this be tested? 
- Checkout the branch and run locally 
- Install dependencies and run yarn start 
- You should see a sample list of items already recorded
